src/Compilers/Core/Portable/NonNullTypesAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace System.Runtime.CompilerServices
+{
+    // PROTOTYPE(NullableDogfood): Remove attribute definition if synthesized by the compiler.
+    internal sealed class NonNullTypesAttribute : Attribute
```

Comment:
Consider doing a linked file here rather than copy/pasting the definition multiple times.